src/Components/CreateTask.js

```diff
@@ -0,0 +1,149 @@
+import { useState } from "react";
+import { Form, Button, Container, Col, Row } from "react-bootstrap";
+import Parse from "parse";
+import TitleForm from "../Components/FormComponents/TitleForm"
+
+export function CreateTask() {
+  console.log("DATE");
+  process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+  const [title, setTitle] = useState();
+  const [responsible, setResponsible] = useState();
+  const [description, setDescription] = useState();
+  const [date, setDate] = useState();
+
+  async function handleUpload(e) {
+    e.preventDefault();
+    console.log("prevented default");
+
+    console.log(responsible);
+    console.log(description);
+
+    const Task = Parse.Object.extend("Task");
+    const newTask = new Task();
+    newTask.set("title", title);
+    newTask.set("responsible", "You");
+    newTask.set("date", date)
+    newTask.set("status", "To do")
+    newTask.set("section", "Culture")
+    newTask.set("description", description);
+
+    try {
+      await newTask.save();
+      alert("succes");
+    } catch (error) {
+      alert(error);
+    }
+  }
+
+  return (
+    <>
```

Comment:
Over 100 lines of JSX in this return statement. That is too much :)
Needs a refactor

Reply:
(I'm talking about making it into smaller components you then reuse)